code/modules/economy/EFTPOS.dm

```diff
@@ -38,7 +38,10 @@
 	R.stamped += /obj/item/stamp
 	R.overlays += stampoverlay
 	R.stamps += "<HR><i>This paper has been stamped by the EFTPOS device.</i>"
-	var/obj/item/smallDelivery/D = new(R.loc)
+	var/obj/item/smallDelivery/D = new(get_turf(loc))
+	if(istype(loc, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = loc
+		D.forceMove(H.back)
```

Comment:
Will have to check if `H.back` is null here as well. Edge case most likely but it's best to be safe. Otherwise, you can also try put it in his hands

Reply:
The code that creates an eftpos, which calls this, is only found in the backpack_contents list for chef outfit, so it might not be necessary to check.

Reply:
Put a check in. Admins may spawn an EFTPOS.

Reply:
Okay, if there is no backpack it'll just spawn on the floor, putting in hands would have the same issue if admin tries to spawn it on the ground, floor is just easier.

Reply:
it'll try to put it in the hands of the `loc`/`human`. If that fails then it'll drop it to the floor. That's how the `put_in_hands` proc works

Reply:
Oh I see, I thought you meant hands in the same way as human/back. Either way should have the intended effect.